fluid/face_detction/train.py

```diff
@@ -37,9 +41,17 @@ def train(args,
     place = fluid.CUDAPlace(0) if args.use_gpu else fluid.CPUPlace()
     exe = fluid.Executor(place)
     exe.run(fluid.default_startup_program())
-    #print(fluid.default_main_program())
-    #print(test_program)
-    #fluid.io.save_persistables(exe, "model")
+
+    # load_model.load_and_set_vars(place)
```

Comment:
The load_mode.py and `vgg/names.map` can be removed, if the model has been saved as Fluid format.

Reply:
Done

Reply:
Also need to remove this line.

Reply:
Done